sass/themes/schemas/components/light/_chip.scss

```diff
@@ -176,6 +177,11 @@ $light-chip: extend(
                 200,
             ),
         ),
+        size: (
+            rem(20px),
+            rem(24px),
+            rem(32px),
```

Comment:
Where do these sizes come from? I couldn't find anything related to size in the theme.